furms-rabbitmq-site-client/src/main/java/io/imunity/furms/rabbitmq/site/client/SiteAgentListenerRouter.java

```diff
@@ -32,12 +32,17 @@
 
 	@RabbitHandler
 	public void receive(Payload<?> payload) {
-		publisher.publishEvent(payload);
+		try {
+			publisher.publishEvent(payload);
+		}catch (Exception e){
+			LOG.error("Received payload cannot be processed {}", payload);
```

Comment:
This can be squeezed in to single error record instead of two log entries, the same applies to the FurmsPayloadConverter